dev/io.openliberty.accesslists.internal/src/com/ibm/ws/tcpchannel/internal/FilterListFastStr.java

```diff
@@ -69,6 +73,10 @@ public boolean buildData(String[] data) {
         for (int i = 0; i < length; i++) {
             if (!addAddressToList(data[i])) {
                 return false;
+            } else {
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.debug(tc, "host added to list of trusted hosts: " + data[i]);
```

Comment:
we should have a more generic message here

Reply:
Yes I saw that.  I had not edited these - apart from merging the different versions that were present into a 'superset' of lines - so that code came from one of the versions. but I will improve it now. It is ALSO used for
exclude lists - so is just plain wrong.

Reply:
Done